packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -1,98 +1,131 @@
+│   ├── <a href="./proofs/interfaces">interfaces</a>: Interface files for src contracts, to avoid unnecessary compilation of contracts
+│   └── <a href="./proofs/utils">utils</a>: Proof dependencies, including the autogenerated deployment summary contracts
+└── <a href="./scripts">scripts</a>: Where the scripts of the projects live
+    ├── <a href="./scripts/json">json</a>: Data cleaning scripts for the output of <a href="./KontrolDeployment.sol">KontrolDeployment.sol</a>
+    ├── <a href="./scripts/make-summary-deployment.sh">make-summary-deployment.sh</a>: Executes <a href="./KontrolDeployment.sol">KontrolDeployment.sol</a>, curates the result and writes the summary deployment contract
+    └── <a href="./scripts/run-kontrol.sh">run-kontrol.sh</a>: Wrapper around the kontrol CLI to run the proofs
+</pre>
 
-### [`proofs`](./proofs) folder
+### Installation
 
-- [`L1CrossDomainMessenger.k.sol`](./proofs/L1CrossDomainMessenger.k.sol): Symbolic property tests for [`L1CrossDomainMessenger`](../../src/L1/L1CrossDomainMessenger.sol)
-- [`L1ERC721Bridge.k.sol`](./proofs/L1ERC721Bridge.k.sol): Symbolic property tests for [`L1ERC721Bridge`](../../src/L1/L1ERC721Bridge.sol)
-- [`L1StandardBridge.k.sol`](./proofs/L1StandardBridge.k.sol): Symbolic property tests for [`L1StandardBridge`](../../src/L1/L1StandardBridge.sol)
-- [`OptimismPortal.k.sol`](./proofs/OptimismPortal.k.sol): Symbolic property tests for [`OptimismPortal`](../../src/L1/OptimismPortal.sol)
-- [`interfaces`](./proofs/interfaces): Files with the signature of the functions involved in the verification effort
-- [`utils`](./proofs/utils): Proof dependencies, including the summary contracts
+1. `cd` to the root of this repo.
+2. Install Foundry by running `pnpm install:foundry`. This installs `foundryup`, the foundry toolchain installer, then installs the required foundry version.
+3. Install Kontrol by running `pnpm install:kontrol`. This installs `kup`, the package manager for RV tools, then installs the required kontrol version.
+4. Install Docker.
 
-### [`scripts`](./scripts) folder
+## Usage
 
-- [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh): Executes [`KontrolDeployment.sol`](./KontrolDeployment.sol), curates the result and writes the summary deployment contract
-- [`run-kontrol.sh`](./scrpts/run-kontrol.sh): Proof execution script
-- [`json`](./scripts/json): Data cleaning scripts for the output of [`KontrolDeployment.sol`](./KontrolDeployment.sol)
+Verifying proofs has two steps: build, and execute.
 
-## Verification execution
+### Build Deployment Summary
 
-The verification execution consists of two steps, although the first step may be omitted to use the committed version. There's one script to run per step. These scripts should be run from the [`contracts-bedrock`](../../) directory.
+First, generate a deployment summary contract from the deploy script in [`KontrolDeployment.sol`](./KontrolDeployment.sol) by running the following command:
```

Comment:
the KontrolDeployment.sol link target is broken here and elsewhere

Reply:
Thanks, fixed in 1861fde98